pkg/liqo-controller-manager/resource-request-controller/offerUpdater.go

```diff
@@ -175,12 +136,42 @@ func (u *OfferUpdater) createOrUpdateOffer(cluster discoveryv1alpha1.ClusterIden
 		return true, err
 	}
 	klog.Infof("%s -> %s Offer: %s/%s", u.homeCluster.ClusterName, op, offer.Namespace, offer.Name)
-	return true, nil
+	return false, nil
 }
 
+// NotifyChange is used by the ResourceReader to notify that resources were added or removed.
+// It checks if any resources have changed by at least a set percentage since we last updated a ResourceOffer, and if so
+// it triggers a new update.
+func (u *OfferUpdater) NotifyChange() {
+	for clusterID := range u.currentResources {
+		if u.isAboveThreshold(clusterID) {
+			var clusterIdentity discoveryv1alpha1.ClusterIdentity
+			cluster, err := foreigncluster.GetForeignClusterByID(context.Background(), u.client, clusterID)
```

Comment:
Would avoid retrieving this here, to avoid performing potentially useless operations.